modules/documents/client/directives/documents.manager.edit.directive.js

```diff
@@ -37,43 +36,49 @@ angular.module('documents')
 							$scope.originalName = obj.displayName || obj.documentFileName || obj.internalOriginalName;
 							$scope.doc = obj;
 							// any dates going to the datepicker need to be javascript Date objects...
-							$scope.doc.documentDate = _.isEmpty(obj.documentDate) ? null : moment(obj.documentDate).toDate();
-							$scope.datePicker = {
-								opened: false
-							};
-							// Used to remove keywords from the document model
-							$scope.removeKeyword = function (keyword) {
-								_.remove($scope.doc.keywords, function (word) {
-									return word === keyword;
-								});
-							};
+							function prepDate(model) { return _.isEmpty(model) ? null : moment(model).toDate();}
+							$scope.doc.documentDate = prepDate(obj.documentDate);
+							$scope.doc.dateUploaded = prepDate(obj.dateUploaded);
+							if ($scope.doc.inspectionReport) {
+								$scope.doc.inspectionReport.dateReportIssued 	= prepDate(obj.inspectionReport.dateReportIssued);
+								$scope.doc.inspectionReport.dateResponse 	= prepDate(obj.inspectionReport.dateResponse);
+								$scope.doc.inspectionReport.dateFollowUp 	= prepDate(obj.inspectionReport.dateFollowUp);
+							}
+
+							$scope.datePicker = { opened: false };
```

Comment:
These are unecessary, you just need to toggle the var in the html.  See comment below.